base/steer/FairRootManager.cxx

```diff
@@ -103,7 +105,7 @@ FairRootManager::FairRootManager()
     , fEntryNr(0)
     , fListFolder(0)
     , fSource(0)
-    , fSourceChain(new TChain(GetTreeName(), Form("/%s", GetFolderName())))
+    , fSourceChain(0)   // 2022.01.11: moved the creation of fSourceChain to InitSource
```

Comment:
Is the comment really needed?

What about moving this into the header like this?
```c++
TChain* fSourceChain = nullptr;
```